data-structures-linked-lists/remove-tail.js

```diff
@@ -0,0 +1,8 @@
+/* exported removeTail */
+
+function removeTail(list) {
+  if (list.next === null) {
+    return list;
+  }
+  list.next.next.next = null;
```

Comment:
This will specifically work to pass the tests but won't work with a list of any size, you should be using a `while` loop to traverse the list until you find the node before the tail then remove the tail.